### PR TITLE
server/bangs: Added archlinux, imdb, python and stackoverflow bangs

### PR DESCRIPTION
**Pull request by gkbrk**

### CODE REVIEW

**Comment by sylvinus** [comment]
Hi @gkbrk! Thanks for the patch :)

I think we need to have a broader discussion around our policy for including and naming bang codes. 

As an example, adding `!imdb` seems like a no-brainer, but on what grounds should we give `!aw` to wiki.archilinux.org?

It would be interesting to research duckduckgo's policies and other organizations that had to face a similar issue!

**Comment by gkbrk** [comment]
I actually got `!aw` from DuckDuckGo, but I agree that single or two-lettered bangs should be allocated carefully. I can rename it `!archwiki` or keep it temporarily with a comment to discuss it later.

**Comment by sylvinus** [comment]
Ok. Another question raised by `!python`: should we point to python 3 or python 2? I don't want to start a debate on python 3 adoption and it's admittedly a small detail, but I'm interested in the resolution process itself.

**Comment by gkbrk** [comment]
Hmm, I think `!python` and `!py` for Python 3 and `!python2` and `!py2` for Python 2 should cover most cases. Adding `!py3` and `!python3` for the sake of completeness might be a good idea too.

**Comment by sylvinus** [comment]
I'm still thinking about this ; should we bootstrap a larger database of bangcodes semi-automatically or wait for people to send pull requests for their favorite ones?

**Comment by hjacobs** [comment]
@sylvinus adding bangs "on-demand" (waiting for pull requests) is fine I think.

File: server/bangs.json [hunk 1]
```diff
@@ -26,8 +37,32 @@
...
+  },
+
 
+  "py": {
+    "any": "https://docs.python.org/3/search.html?q={{{s}}}"
+  },
+  "python": {
+    "any": "https://docs.python.org/3/search.html?q={{{s}}}"
+  },
+  "pip": {
```
Comment (hjacobs) [inline server/bangs.json:53]: "pip" is just one client tool (but having the bang is fine), let's add "pypi" too

**Comment by sylvinus** [comment]
ok! @gkbrk can you update the PR with everything that was said here?

**Comment by gkbrk** [comment]
I'll update the PR today after school.

**Comment by gkbrk** [comment]
Removed `!aw` for Arch Wiki, added `!pypi` for PyPi.

**Comment by sylvinus** [comment]
Great, thanks!

**Pull request merged**